app/scripts/lib/setupSentry.js

```diff
@@ -86,7 +86,9 @@ function getClientOptions() {
     integrations: [
       Sentry.dedupeIntegration(),
       Sentry.extraErrorDataIntegration(),
-      Sentry.browserTracingIntegration(),
+      Sentry.browserTracingIntegration({
+        enableLongAnimationFrame: true,
```

Comment:
Nit: I see that [the blog post that introduced this feature](https://sentry.io/changelog/long-animation-frames-in-javascript-sdk/) claims it will be enabled by default in version 8.26, and we [just updated to version 8.33.1](https://github.com/MetaMask/metamask-extension/pull/27620). Perhaps we can omit this?

Reply:
Done.